docker-compose.yaml

```diff
@@ -0,0 +1,23 @@
+#https://docs.docker.com/registry/deploying/#deploy-your-registry-using-a-compose-file
+
+version: '3'
+
+services:
```

Comment:
We need to add nginx here, and have it use your config.

Reply:
We also need to add `certbot` to automate renewing certificates.

Reply:
https://github.com/Seneca-CDOT/telescope/blob/master/docker/production.yml#L150-L159